### PR TITLE
add Cisco C9500-12Q

### PR DESCRIPTION
**Pull request by christian-boesch**

### CODE REVIEW

File: device-types/Cisco/C9500-12Q.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+manufacturer: Cisco
+model: Catalyst 9500-12Q
+slug: cisco-c9500-12q
+part_number: C9500-12q
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/Cisco/C9500-12Q.yaml:7]: Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Comment (danner26) [inline device-types/Cisco/C9500-12Q.yaml:7]: Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

File: device-types/Cisco/C9500-12Q.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    type: rj-45
+  - name: usb
+    type: usb-mini-b
+power-ports:
+  - name: PS-0
+    type: iec-60320-c14
+    maximum_draw: 950
+  - name: PS-1
+    type: iec-60320-c14
+    maximum_draw: 950
```
Comment (danner26) [inline device-types/Cisco/C9500-12Q.yaml:20]: Are the PSUs in this unit modular? If so, you should create `module-bays` for the PSUs and then the associated modular PSUs should be added to the module-type folder if they do not already exist

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Cisco/C9500-12Q.yaml (hunk 1 above)
Comment (christian-boesch) [reply device-types/Cisco/C9500-12Q.yaml:10]: done

Comment (christian-boesch) [reply device-types/Cisco/C9500-12Q.yaml:10]: done

File: device-types/Cisco/C9500-12Q.yaml (hunk 2 above)
Comment (christian-boesch) [reply device-types/Cisco/C9500-12Q.yaml]: done

**Review by danner26 — APPROVED** [review]

**Pull request merged**